patches/components-policy-resources-policy_templates.json.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/components/policy/resources/policy_templates.json b/components/policy/resources/policy_templates.json
+index 5238b76ba5466b3015157bc33e4efe13a2213418..382aeede0591a604f48d85cac70389ce38cf619d 100644
+--- a/components/policy/resources/policy_templates.json
++++ b/components/policy/resources/policy_templates.json
+@@ -1,4 +1,4 @@
+-{
++__import__('policy_source_helper').BravePolicies() + {
```

Comment:
will it work like this?
```
__import__('policy_source_helper').BravePolicies() +
{
```
or
```
__import__('policy_source_helper').BravePolicies() + \
{
```

Reply:
This PR will likely have to be re-done in its entirety because upstream just completely changed their implementation. Settings used to be defined in a single .json file. Now it's [a set of directories and .yaml files](https://github.com/chromium/chromium/blob/main/docs/enterprise/add_new_policy.md).